api/src/main/java/com/epam/pipeline/dao/monitoring/metricrequester/CPURequester.java

```diff
@@ -75,6 +75,7 @@ public SearchRequest buildRequest(final Collection<String> resourceIds, final Lo
     @Override
     public Map<String, Double> parseResponse(final SearchResponse response) {
         return ((Terms)response.getAggregations().get(AGGREGATION_POD_NAME)).getBuckets().stream()
+                .filter(b -> Double.isFinite(((Avg) b.getAggregations().get(AVG_AGGREGATION + USAGE_RATE)).getValue()))
```

Comment:
Btw, there is a handy null-safe `AbstractMetricRequester.doubleValue` function which can be used to reduce boilerplate code. Moreother it already filters out infinite numbers.
```
doubleValue(aggregations(b), AVG_AGGREGATION + NODE_UTILIZATION))
```